Bottom: 7b1e5969058dc82b9e4c4cc96dd324032165ce4d
Top:    9b38dbe489f6d1e4602f1f9bb69a7ed4268fdef7
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-26 03:06:04 +0100

drc: HACK HACK HACK


---

diff --git a/src/Makefile.am b/src/Makefile.am
index 61eacba..8f698f4 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -283,6 +283,7 @@ libgtk_a_CPPFLAGS = -I./hid/gtk
 LIBGTK_SRCS = \
 	dolists.h \
 	hid/hidint.h \
+	hid/gtk/bga-wizard.c \
 	hid/gtk/gtkhid-main.c \
 	hid/gtk/gtkhid.h \
 	hid/gtk/gui.h \
